src/helpers/chevrotain.ts

```diff
@@ -0,0 +1,12 @@
+import { createToken, ITokenConfig } from 'chevrotain';
+
+const createKeyword = ({ name, pattern, ...props }: ITokenConfig) =>
+  createToken({
+    name,
+    pattern: pattern || new RegExp(name, 'i'),
+    ...props,
+  });
+
+export default {
```

Comment:
we generally don't use default exports, so you can export the `createKeyword` function as a named export